awx/settings/development.py

```diff
@@ -53,7 +53,13 @@
 
 # celery is annoyingly loud when docker containers start
 LOGGING['loggers'].pop('celery', None)  # noqa
+# avoid awx.main.dispatch WARNING-level scaling worker up/down messages
+LOGGING['loggers']['awx.main.dispatch']['level'] = 'ERROR'  # noqa
+# suppress the spamminess of the awx.main.scheduler and .tasks loggers
+LOGGING['loggers']['awx']['level'] = 'INFO'  # noqa
 
+# allow Django Debug Toolbar to work in the local dev environment
+INTERNAL_IPS = ('127.0.0.1', '172.19.0.1', '172.18.0.1', '192.168.100.1')
```

Comment:
Do we really want to have DDT enabled by default?  iirc, it adds a fair bit of overhead to requests.

Reply:
I obviously strongly think so, but let's see if more people are opposed.

Reply:
All right, pruning it out based on @matburt 's very strong opinion on the matter.